whatwg.org/irc

```diff
@@ -56,7 +56,9 @@
 
 <h3 id="getting-started-with-irc">Getting Started with IRC<a class="self-link" href="#getting-started-with-irc"></a></h3>
 
-<p>The simplest way to get started with IRC, if you are not familiar, is by signing up for a free <a href="https://www.irccloud.com/">IRCCloud</a> account. Once you've done that, you should be logged in to the Freenode server by default. All you'll have to do is join the #whatwg channel. If you tell your browser to use IRCCloud for irc:// links, then just <a href="ircs://irc.freenode.net/whatwg">clicking this link</a> should also take you there.</p>
+<p>The simplest way to get started with IRC, if you are not familiar, is by signing up for a free <a href="https://www.irccloud.com/">IRCCloud</a> account. Once you've done that, you should be logged in to the Freenode server by default. All you'll have to do is join the #whatwg channel. If you tell your browser to use IRCCloud for irc:// links, then just <a href="ircs://irc.freenode.net/whatwg">clicking this link</a> should also take you there. In order to join the channel, you will need to <a href="https://freenode.net/kb/answer/registration">register your nick on Freenode</a>.</p>
+
+<p class="note">In view of ongoing spam messaging on Freenode, we highly suggest <a href="https://freenode.net/kb/answer/usermodes">you set mode</a> to <pre>+r</pre> in order to block unregistered users from sending you messages.</p>
```

Comment:
"you set mode" -> "you set your mode". Maybe also just link "set your mode", not "you".